Rlabkey/R/labkey.experiment.R

```diff
@@ -170,7 +170,7 @@ labkey.experiment.saveBatch <- function(baseUrl=NULL, folderPath, assayConfig =
         params = list()
         params$protocolName = protocolName
     }
-    params$batch = list(runs = ensureNestedList(runList))
+    params$batch = c(batchPropertyList, list(runs = ensureNestedList(runList)))
```

Comment:
should there be a null check before we add the batch properties to the collection?

Reply:
Looks like that isn't needed in R. The following both result in the same output:
`list(runs = "test")`
`c(NULL, list(runs = "test"))`
R looks to drop the NULL from the vector creation.